tests/unit/src/test/resources/semanticTokens/example/Imports.scala

```diff
@@ -0,0 +1,10 @@
+<<package>>/*keyword*/ <<example>>/*namespace*/
+
+<<import>>/*keyword*/ util.{<<Failure>>/*class*/ <<=>>>/*operator*/ NotGood}
```

Comment:
```suggestion
<<import>>/*keyword*/ util.{<<Failure>>/*class*/ <<=>>>/*operator*/ NotGood/*class*/ }
```
seems we are not traversing renames, but this can be done as a follow up.